README.md

```diff
@@ -35,6 +35,12 @@ If pkg-config is being used, it's configuration options will apply.
 
 * `buildtime_bindgen`: Run `bindgen` at build-time to generate bindings using installed headers. Not compatible with the `bundled` feature.
 * `bundled`: Build the bundled version of `libpq` from source.
+  Another option to compile `libpq` from source without enabling `openssl` usage is to add the `pq-src` crate with `disable-openssl` features to 
+  your crate dependencies and not using the `bundled` feature of `pq-sys`:
```

Comment:
This comment should be more explicit about what are the implications of enabling this feature. It should more or less literally say, that this disables the SSL support in libpq, so that you cannot connect to database requiring SSL anymore.

Reply:
Thanks for the suggestion, I reword the comment